modules/benchmark/R/calc.benchmark.R

```diff
@@ -18,18 +18,15 @@ calc.benchmark <- function(settings, con){ #settings file is output from start.b
   
   # Query BRR for benchmark ids
   
-  bm.id <- settings$benchmark$id #maybe something like that?
+  bm.id <- settings$benchmark$id 
+  obvs.id <- settings$benchmark$input_id
```

Comment:
I'm assuming this is the ID of the observed data being benchmarked against? If so, that should be coming from the database, not the settings